cmd/thor/main.go

```diff
@@ -302,6 +309,7 @@ func defaultAction(ctx *cli.Context) error {
 		p2pCommunicator.Communicator(),
 		ctx.Uint64(targetGasLimitFlag.Name),
 		skipLogs,
+		requireTxPriorityFee,
```

Comment:
Do you think it's worth start grouping all the flags/options under one struct like `NodeOption` (something like the [mempool options](https://github.com/vechain/thor/blob/master/txpool/tx_pool.go#L55))? We could group together  
```
...
ctx.Uint64(targetGasLimitFlag.Name),
skipLogs,
requireTxPriorityFee,
...
```
I'm not saying that this needs to be done in this ticket, but maybe it can be done in another one. wdyt?

Reply:
I think it makes total sense, since I will be waiting for more reviewers I think I can just do it in the meantime here, thanks.

Reply:
Done now